cup/README.md

```diff
@@ -0,0 +1,28 @@
+# Cup and Flipt
+
+<div align="center">
+  <img src="../images/cup.svg" alt="CUP" width="240" />
+</div>
+
+## Overview
+
+This application is designed to demonstrate [cup](https://github.com/flipt-io/cup), configured to manage Flipt flag state.
+
+In particular, this application demonstrates how Cup and Flipt can be used together with the Git backend for Flipt.
+
+## Prerequisites
+
+- [Docker](https://docs.docker.com/get-docker/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+- [Cup CLI](https://github.com/flipt-io/cup) cloned and built from source
+
+## Usage
+
+1. `cd` into this directory (e.g. `cd cup`)
+1. Run `docker compose up --build`
+1. Run `cup defs` to connect to `cupd` and see available resources
+1. Run `cup get flags` to see currently served flags
```

Comment:
is the thinking that we'll expand this to allow them to change the flag like you do in the demo?

Reply:
Yeah, I was thinking that would come when we add a full UI walkthrough like the other labs